qctrlopencontrols/qiskit/quantum_circuit.py

```diff
@@ -267,12 +267,13 @@ def convert_dds_to_quantum_circuit(
         gate_time=gate_time,
         unitary_time=unitary_time)
 
-    for qubit in target_qubits:
-        quantum_circuit.u3(pre_post_gate_parameters[0], #pylint: disable=no-member
-                           pre_post_gate_parameters[1],
-                           pre_post_gate_parameters[2],
-                           quantum_registers[qubit])
-        quantum_circuit.barrier(quantum_registers[qubit]) #pylint: disable=no-member
+    if pre_post_gate_parameters is not None:
+        for qubit in target_qubits:
+            quantum_circuit.u3(pre_post_gate_parameters[0], #pylint: disable=no-member
```

Comment:
For my education, why's the disable=no-member tag necessary? Is there something in quantum_circuit we need to fix?

Reply:
@charmasaur - no it is a strange way how these are added to Qiskit library (the IBM One). This one might shed some light - https://quantumcomputing.stackexchange.com/questions/4962/how-to-fix-e1101instance-of-quantumcircuit-has-no-h-member-and-e1101in

Reply:
Gotcha, thanks. Python is scary :S